### ISSUE TITLE
suggestions: in Argument attribute

### ISSUE THREAD

**Issue by tacman** [zenstruck/console-extra#63]
Is that a v2 thing?  The README for v1 shows an example of that, but it's failing.

Happy to test v2 if it's ready for testing.

**Comment by kbond** [comment]
Are you using dev? I haven't tagged a release yet

**Comment by tacman** [comment]
Ah, I see -- 1.x is dev, I thought it was the latest release.  I'll try that .

**Comment by kbond** [comment]
I'll make a release soon. I want to test the deprecation flow in one of my apps first.

**Comment by tacman** [comment]
Sigh.  Not sure how to do that.

```
composer req zenstruck/console-extra:1.x
./composer.json has been updated
```

Didn't get me the dev version -- what's the correct way to do that?

**Comment by kbond** [comment]
`composer require zenstruck/console-extra:"^1.0@dev"` should do the trick

**Comment by kbond** [comment]
FYI, I've released 1.4.0
